### PR TITLE
StateView::id() -> StateViewId

### PR DESCRIPTION
**Pull request by msmouse**
## Motivation

For logging and debugging purposes.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

yes
## Test Plan
compiling

### CODE REVIEW

**Comment by dariorussi** [comment]
this seems good to me but I think others here should comment, @msmouse let me know if you need approval.
I have added @runtian-zhou to the review in case he has comments

File: storage/state-view/src/lib.rs [hunk 1]
```diff
@@ -6,12 +6,18 @@
...
 use anyhow::Result;
-use libra_types::access_path::AccessPath;
+use libra_crypto::HashValue;
+use libra_types::{access_path::AccessPath, transaction::Version};
 
 /// `StateView` is a trait that defines a read-only snapshot of the global state. It is passed to
 /// the VM for transaction execution, during which the VM is guaranteed to read anything at the
 /// given state.
 pub trait StateView {
+    /// For logging and debugging purpose, identifies what's this view is for.
```
Comment (wqfish) [inline storage/state-view/src/lib.rs:16]:
```suggestion
    /// For logging and debugging purpose, identifies what this view is for.
```

File: storage/storage-interface/src/state_view.rs [hunk 2]
```diff
@@ -26,6 +26,9 @@ use std::{
 /// `VerifiedStateView` is like a snapshot of the global state comprised of state view at two
 /// levels, persistent storage and memory.
 pub struct VerifiedStateView<'a> {
+    /// For logging and debugging purpose, identifies what's this view is for.
```
Comment (wqfish) [inline storage/storage-interface/src/state_view.rs:29]: same here

**Pull request closed without merging**